More/EAV/Tests/ValueTest.php

```diff
@@ -40,7 +40,6 @@ public function testSetGetAttribute()
         $value->setName('Silver');
         $this->assertSame($attribute, $value->getAttribute());
 
-        $value->setAttribute($attribute);
-        $this->assertSame($attribute, $value->getAttribute());
+        $this->assertSame($attribute, $value->setAttribute($attribute)->getAttribute());
```

Comment:
I prefer to keep it as it was...  I think it's now harder to understand by reason of implicit second argument which has overloaded action.